src/disaster_vehicle.cpp

```diff
@@ -456,7 +456,7 @@ static bool DisasterTick_Aircraft(DisasterVehicle *v, uint16 image_override, boo
 			DestructIndustry(i);
 
 			SetDParam(0, i->town->index);
-			AddTileNewsItem(news_message, NT_ACCIDENT, v->dest_tile);
+			AddIndustryNewsItem(news_message, NT_ACCIDENT, (IndustryID)v->dest_tile);
```

Comment:
better use `i->index` ? As `i->town->index` is still valid, I assume so is `i->index`. But that is an assumption :D

Reply:
Owh, it used to be `i->index`. Seems we should just revert the change I made in dff7ee8e1c68199a0dc105e82e588ac097543645 as the correct solution. I think it just got caught in my search for `->index`, without realising this was the industry, instead of the vehicle.

Reply:
I thought about it, but figured it might be slightly cleaner to use the variable that we use to get the `Industry` object in the first place, a couple of lines further up ¯\\\_(ツ)_/¯